public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -418,7 +420,7 @@ export const DashNav = React.memo<Props>((props) => {
 
   return (
     <PageToolbar
-      pageIcon={isFullscreen ? undefined : 'apps'}
+      pageIcon={isFullscreen || !title ? undefined : 'apps'}
```

Comment:
there's no property in dashboard metadata in order not to show the pageIcon. I think not having a title could be a reasonable thing, isn't it? Maybe something to ask to ux?

Reply:
Why not show title?